tools/isobuild/bundler.js

```diff
@@ -628,7 +636,7 @@ class File {
   }
 
   sri() {
-    if (! this._sri) {
+    if (! this._sri && !this._skipSri) {
```

Comment:
I think this is wrong place to put this. I think if you call `sri()`, sha512 should always be computed.

Reply:
You should wrap `manifestItem.sri = file.sri()` with `_skipSri` check.